conans/client/conf/__init__.py

```diff
@@ -71,6 +71,13 @@
             threads: [None, posix, win32] #  Windows MinGW
             exception: [None, dwarf2, sjlj, seh] # Windows MinGW
             cppstd: [None, 98, gnu98, 11, gnu11, 14, gnu14, 17, gnu17, 20, gnu20]
+        msvc:
+            version: ["19.0",
```

Comment:
are we going to define the previous versions as well (pre-19.0)?

Reply:
Not at the moment, we are starting with VS 2015 for this proposal, and once the functionality is stabilized a bit, we will get the feedback from the community/tribe regarding the supported versions.